{
    "cve": {
        "data_type": "CVE",
        "data_format": "MITRE",
        "data_version": "4.0",
        "CVE_data_meta": {
            "ID": "CVE-2020-12050",
            "ASSIGNER": "cve@mitre.org"
        },
        "problemtype": {
            "problemtype_data": [
                {
                    "description": []
                }
            ]
        },
        "references": {
            "reference_data": [
                {
                    "url": "http://www.ch-werner.de/sqliteodbc/",
                    "name": "http://www.ch-werner.de/sqliteodbc/",
                    "refsource": "MISC",
                    "tags": []
                },
                {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1825762",
                    "name": "https://bugzilla.redhat.com/show_bug.cgi?id=1825762",
                    "refsource": "CONFIRM",
                    "tags": []
                },
                {
                    "url": "https://sysdream.com/news/lab/",
                    "name": "https://sysdream.com/news/lab/",
                    "refsource": "MISC",
                    "tags": []
                }
            ]
        },
        "description": {
            "description_data": [
                {
                    "lang": "en",
                    "value": "SQLiteODBC 0.9996, as packaged for certain Linux distributions as 0.9996-4, has a race condition leading to root privilege escalation because any user can replace a /tmp/sqliteodbc$$ file with new contents that cause loading of an arbitrary library."
                }
            ]
        }
    },
    "configurations": {
        "CVE_data_version": "4.0",
        "nodes": []
    },
    "impact": {},
    "publishedDate": "2020-04-30T17:15Z",
    "lastModifiedDate": "2020-04-30T17:17Z"
}